app/graphql/locale.py

```diff
@@ -0,0 +1,8 @@
+from typing import NewType
+
+import strawberry
+
+Locale = strawberry.scalar(
```

Comment:
do we need this one? can we just use str instead?

Reply:
My understanding is that this is a way to future-proof this field, by preventing clients from assuming this is a string. In the future we might want to change the type to an Enum. We do the same for [Url](https://studio.apollographql.com/graph/pocket-client-api/schema/reference/scalars?query=Url&variant=current).

I'll ask advice from the backend team whether this is best practice.

Reply:
I looked for some more examples and found `CollectionsFiltersInput`, which was recently added and uses `language: String`. I removed the Locale scalar based on that.